test/utils/deployment.ts

```diff
@@ -0,0 +1,121 @@
+import { logUtils as log } from '@0x/utils';
+import { ChildProcessWithoutNullStreams, spawn } from 'child_process';
+import { resolve as resolvePath } from 'path';
+
+const apiRootDir = resolvePath(`${__dirname}/../../../`);
+
+let yarnStartProcess: ChildProcessWithoutNullStreams;
+
+/**
+ * The configuration object that provides information on how verbose the logs
+ * should be.
+ * @param shouldPrintApiLogs Whether or not the 0x-api logs should be surfaced.
+ * @param shouldPrintDependencyLogs Whether or not the 0x-api's dependencies
+ *        should surface their logs.
+ * TODO(jalextowle): It would be a good improvement to be able to specify log
+ * files where the logs should actually be written.
```

Comment:
👍

Reply:
As of now, does it output the logs to the same stdout as the test results?

Reply:
Yep. Until the log output can be redirected to a file, I've been seeing this as a feature that developers will use while writing and debugging tests, but not as much afterwards.